bokeh/core/properties.py

```diff
@@ -1543,33 +1541,16 @@ class Datetime(PropertyDescriptor):
 
     """
 
+    _underlying_types = datetime_types
+
     def __init__(self, default=datetime.date.today(), help=None):
         super(Datetime, self).__init__(default=default, help=help)
 
     def validate(self, value):
         super(Datetime, self).validate(value)
 
-        datetime_types = (datetime.datetime, datetime.date)
-        try:
-            import numpy as np
-            datetime_types += (np.datetime64,)
-        except (ImportError, AttributeError) as e:
-            if e.args == ("'module' object has no attribute 'datetime64'",):
-                import sys
-                if 'PyPy' in sys.version:
-                    pass
-                else:
-                    raise e
-            else:
-                pass
```

Comment:
Someone has to explain this code to me. I'm not sure why we have to make PyPy involved in this.